arch/ARM/Nordic/drivers/nrf51-uart.adb

```diff
@@ -0,0 +1,184 @@
+with NRF51_SVD.UART;  use NRF51_SVD.UART;
+
+package body nRF51.UART is
+   
+   procedure Enable (This : in out UART) is
```

Comment:
In the body of a package, each subprogram implementation should have what we call a subprogram box.
It is a comment formatted like this:

```
   ----------------------------
   -- Name_Of_The_Subprogram --
   ----------------------------
```
In GNAT Programing Studio, you can define a shortcut to create the subprogram box (I use ctrl+b).